.github/README.md

```diff
@@ -0,0 +1,51 @@
+# [Oppia-Android](https://www.oppia.org)
+
+Oppia-Android is a mobile application version of [Oppia](https://www.oppia.org). It is an Android application that enables anyone to easily learn through interactive activities (called 'explorations'). These activities simulate a one-on-one conversation with a tutor, making it possible for students to learn by doing while getting feedback.
+
+Moreover, users can create their profile, practice a set of skills, play with explorations, review previously finished skills and also track their progress.
+
+Oppia-Android app is written in Kotlin and Python, and is built on top of Google App Engine.
+
+  * [Oppia.org community site](https://www.oppia.org)
+  * [User Documentation](https://oppia.github.io/)
```

Comment:
Please remove; this link is not relevant to this app.

Reply:
Removed.